src/takescreens.py

```diff
@@ -453,7 +453,11 @@ async def dvd_screenshots(meta, disc_num, num_screens=None, retry_cap=None):
             width = float(track.width)
             height = float(track.height)
             frame_rate = float(track.frame_rate)
-    if par < 1:
+    # Use tolerance-based comparison to avoid scaling when PAR is very close to 1.0
+    # This prevents issues like 1920x1040 becoming 1924x1040 when PAR is 1.002
```

Comment:
IMO, dvd's should always have scaling applied, since they will not suffer the same issue as described in your commentary. And the predominant issue seems to stem from poor webdl handling.

Your CR change also applied to the pr.